### ISSUE TITLE
Stack overflow in RootStackRouter popUntilRoot()

### ISSUE THREAD

**Issue by DHUKK** [Milad-Akarie/auto_route_library#401]
The following code:
`@override
 void popUntilRoot() => popUntilRoot();`
Is causing infinite recursion + stack overflow in:  
- root_stack_router.dart 
  - class RootStackRouter 

I would imagine you would want something like the following:
`void popUntilRoot() => _router.popUntil((route) => route.isFirst);`

## Deps:
```
dependencies:
  flutter_web_plugins:
    sdk: flutter
  flutter:
    sdk: flutter
  auto_route: ^1.0.0
  firebase_core: ^1.0.1
  firebase_auth: ^1.0.1
  flutter_riverpod: ^0.13.1


  cupertino_icons: ^1.0.2

dev_dependencies:
  flutter_test:
    sdk: flutter
  auto_route_generator: ^1.0.0
  build_runner:
```

**Comment by Milad-Akarie** [comment]
@DHUKK  ya this is just untested refactored code 
it should be 
```
@override 
void popUntilRoot() => _router.popUntilRoot();
```

That's an easy fix. Thanks for reporting.

**Comment by Milad-Akarie** [comment]
Fixed in auto_route 1.0.1
